saskatoon/harvest/permissions.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.permissions import IsAuthenticated
+from django.http import HttpRequest
+
+
+class IsCoreOrAdmin(IsAuthenticated):
+    message = "Viewing this page is restricted to core and admin users."
+
+    def has_permission(self, request: HttpRequest, view):
+        return request.user.groups.filter(name__in=["core", "admin"]).exists()
```

Comment:
@mina-andajos since we are subclassing `IsAuthenticated` it is important to first check the user is logged in, otherwise i believe you will get an error because Anonymous user has no groups.
e.g.:
```
        if super().has_permission(request, view):
            return request.user.groups.filter(name__in=["core", "admin"]).exists()
        return False
```